### PR TITLE
docs(Cwgo):update db tutorials version

### PR DESCRIPTION
**Pull request by yuxialuozi**
#### What type of PR is this?
docs
#### Check the PR title.
update db tutorials version

#### (Optional) Translate the PR title into Chinese.

更新了DB的文档
#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:An sql_dir parameter has been added, and we can now specify a sql file or directory
zh(optional): 增加了一个sql_dir参数，现在可以指定一个sql文件或者目录

### CODE REVIEW

File: content/zh/docs/cwgo/tutorials/db/_index.md [hunk 1]
```diff
@@ -56,8 +57,9 @@ OPTIONS:
    --model_pkg value                  指定 model 的包名
    --nullable                         指定生成字段是否为指针当字段为 nullable，默认为 false
    --signable                         指定字段是否检测整型列 unsigned 类型来调整生成相应的数据类型，默认为 false
-   --type_tag                         指定字段是否生成 gorm 的 type tag，默认为 false
+   --type_tag                         指定字段是否生成 gorm 的 type tag，默认为 false         
    --index_tag                        指定字段是否生成 gorm 的 index tag，默认为 false
+   --sql_dir value                    指定一个sql文件或者目录
```
Comment (li-jin-gou) [inline content/zh/docs/cwgo/tutorials/db/_index.md:62]: 使用 sql_dir 时 dsn 参数失效没有说

Comment (yuxialuozi) [reply content/zh/docs/cwgo/tutorials/db/_index.md]: 更改完毕

**Comment by GuangmingLuo** [comment]
Hi, there is no need to change `*` to -, it will both work in markdown. Please revert those changes.

File: content/zh/docs/cwgo/tutorials/db/_index.md [hunk 2]
```diff
@@ -37,6 +37,7 @@ OPTIONS:
    --signable                         Specify detect integer field\'s unsigned type, adjust generated data type (default: false)
    --type_tag                         Specify generate field with gorm column type tag (default: false)
    --index_tag                        Specify generate field with gorm index tag (default: false)
+   --sql_dir value                    Specify a sql file or directory
```
[the remark this answers is not in the document]
Comment (yuxialuozi) [reply content/zh/docs/cwgo/tutorials/db/_index.md]: 修改完毕

**Review by li-jin-gou — APPROVED** [review]

**Pull request merged**